### PR TITLE
On filtering months in FTE page table is available

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #2365 

### Description
On filtering months in FTE page for Projects table is available for all months.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by gautam071** [comment]
@Sachinbisht27 Displaying the table is part of the requirement 
but the table needs to calculate all the details according to the filtered month

**Comment by gautam071** [comment]
Please make the changes accordingly

File: Modules/EffortTracking/Resources/views/show.blade.php [hunk 1]
```diff
@@ -67,7 +67,6 @@
         @endif
         </div>
     </div>
-    @if($totalMonths === 0)
```
Comment (gautam071) [inline Modules/EffortTracking/Resources/views/show.blade.php:70]: why did you remove this code?

File: Modules/EffortTracking/Resources/views/show.blade.php [hunk 2]
```diff
@@ -118,23 +117,22 @@
...
+                                <td>{{ $teamMember->getCurrentExpectedEffortAttribute($startDate, $endDate) }}</td>
+                                <td>{{ $teamMember->getExpectedEffortTillTodayAttribute($startDate, $endDate) }}</td>
+                                <td>{{ $teamMember->getExpectedEffortTillTodayAttribute($startDate, $endDate) - $teamMember->getCurrentActualEffortAttribute($startDate, $endDate) }}
                                 </td>
-                                <td class="{{ $teamMember->velocity >= 1 ? 'text-success' : 'text-danger' }}">
-                                    {{ $teamMember->velocity }}</td>
-                                <td>{{ $teamMember->fte }}</td>
+                                <td class="{{ $teamMember->getVelocityAttribute($startDate, $endDate) >= 1 ? 'text-success' : 'text-danger' }}">
+                                    {{ $teamMember->getVelocityAttribute($startDate, $endDate) }}</td>
+                                <td>{{ $teamMember->getFteAttribute($startDate, $endDate) }}</td>
```
Comment (gautam071) [inline Modules/EffortTracking/Resources/views/show.blade.php:128]: Looks like the function is called in multiple places can we assign a variable for the function and call it

**Review by gautam071 — CHANGES_REQUESTED** [review]
Left some comments

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/EffortTracking/Resources/views/show.blade.php:70]: if  i am putting @if($totalMonths === 0) there the table is only working for current month and if i am putting @if($totalMonths === 0 || $totalMonths != 0) then the table working for both current and previous months also.
But, i thought that if we are putting total month ($totalMonths === 0 || $totalMonths != 0) what is the need of this $totalMonths now.
that's why i removed it...
Should i have to add the condition for ==0 and != 0?

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 2 above)
Comment (Sachinbisht27) [reply Modules/EffortTracking/Resources/views/show.blade.php:128]: *send

Comment (Sachinbisht27) [reply Modules/EffortTracking/Resources/views/show.blade.php:128]: @gautam071 after doing suggested changes i am not getting the data as expected.

**Comment by Sachinbisht27** [comment]
@gautam071 on adding variables over attribute functions in show.blade not getting correct values in table.

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 1 above)
Comment (gautam071) [reply Modules/EffortTracking/Resources/views/show.blade.php:70]: @Sachinbisht27 
`@if($totalMonths === 0 || $totalMonths != 0)`

it means `$totalMonths` if it is zero and not zero it should run right. 
Then what is the use of the check?

**Review by gautam071 — CHANGES_REQUESTED** [review]

File: Modules/EffortTracking/Resources/views/show.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/EffortTracking/Resources/views/show.blade.php:70]: yes, we have to run the table for both $totalMonths zero and not zero.  i thought that if we are putting ($totalMonths === 0 || $totalMonths != 0) what is the need of this. that's why i previously removed it.

**Review by gautam071 — APPROVED** [review]
Looks good to me

**Pull request merged**